### ISSUE TITLE
[Accessibility- information structuring C9.3] Use of lists

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#2915]
**Environment:** https://whitepaper-ft.meeds.io/

**Step of reproduction:**
Access to the left menu

**Current behavior:**
All elements (Pages and spaces) are not structured by lists

**Expected behavior:**
All left menu items should be structured into lists to allow quick consultation of content => Use ul/li lists to structure sequences of elements having the same nature, as well as for successive links

**Debate topic:**
Are there other places where the use of lists is necessary?
